dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/keyjsonvalue/hibernate/HibernateKeyJsonValueStore.java

```diff
@@ -56,12 +59,29 @@
     public List<String> getKeysInNamespace( String namespace )
     {
         String hql = "select key from KeyJsonValue where namespace = :namespace";
-        
+
         return getQuery( hql ).setString( "namespace", namespace ).list();
     }
 
     @Override
     @SuppressWarnings( "unchecked" )
+    public List<String> getKeysInNamespace( String namespace, Date lastUpdated )
+    {
+        String hql = "select key from KeyJsonValue where namespace = :namespace";
+        if(lastUpdated != null) {

```

Comment:
Please apply the DHIS 2 code style, e.g. 

if ( lastUpdated != null )
{
  ..
}